prestodb/spark-base/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM eclipse-temurin:11-jdk
+LABEL author="Nathaniel Vala" email="nathanielvala@hotmail.com"
```

Comment:
use your account, and in readme, mentioning the original repo and author

Reply:
Fixed